ballerina/main.bal

```diff
@@ -0,0 +1,126 @@
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// This block mentions the licensing terms under which the file is provided.
+
+// You may obtain a copy of the License at
+// This line provides the URL where you can find the full text of the Apache License, Version 2.0.
+
+// http://www.apache.org/licenses/LICENSE-2.0
+// This line gives the specific location where the License can be obtained.
+
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations
+// under the License.
+// These lines describe the conditions under which the software is distributed and disclaim any warranties or conditions.
+
+import ballerina/file;
+import ballerina/http;
+import ballerina/io;
+import ballerina/jballerina.java;
+import ballerina/regex;
+import ballerinax/googleapis.sheets as gsheets;
+
+configurable string pdfFilePath = ?;
+configurable string fontFilePath = ?;
+configurable string clientId = ?;
+configurable string clientSecret = ?;
+configurable string refreshToken = ?;
+configurable string refreshUrl = ?;
+configurable string spreadsheetId = ?;
+
+
+
+gsheets:Client spreadsheetClient = check new ({
+    auth: {
+        clientId,
+        clientSecret,
+        refreshToken,
+        refreshUrl
+    }
+});
+
+type ID record {
+    string id;
+};
+
+string file_path = "";
+
+public function PDFGenerator(handle inputFilePath, handle replacement, handle font_type, int fontsize, int centerX, int centerY, handle fontFilePath, handle outputFileName) = @java:Method {
+    'class: "org.PDFCreator.PDFGenerator",
+    name: "pdf"
+} external;
+
+public function certicateGeneration(string inputFilePath, string fontFilePath, string checkID, string sheetName) returns error? {
+    gsheets:Column col = check spreadsheetClient->getColumn(spreadsheetId, sheetName, "C");
+    int i = 1;
+    while i < col.values.length() {
+        gsheets:Row row = check spreadsheetClient->getRow(spreadsheetId, sheetName, i);
+        if row.values[3].toString() == checkID {
+            string replacement = col.values[i].toString();
+            string font_type = row.values[6].toString();
+            io:println(font_type);
+            string fontsizeStr = row.values[7].toString();
+            string centerXstr = row.values[4].toString();
+            string centerYstr = row.values[5].toString();
+            int|error f = int:fromString(fontsizeStr);
+            int|error x = int:fromString(centerXstr);
+            int|error y = int:fromString(centerYstr);
+            int fontsize = 16;
+            int centerX = -1;
+            int centerY = 250;
+            if (f is int) {
+                fontsize = f;
+            }
```

Comment:
You can use `check` when converting the string value to an integer. `int f = check int:fromString(fontsizeStr);`. Then you wouldn't need to explicitly check for the type here.